### PR TITLE
Update creation model v0.9.0 in MTS

### PR DESCRIPTION
**Pull request by galinatremper**
- Added updated mapping.json (publication author, journal, year)
- Closes GDEV-1148

### CODE REVIEW

**Comment by KerstenBreuer** [comment]
Hi @galinatremper, I fixed the template-related checks. Only the tests keep failing.

File: metadata_transpiler_service/creation_models.py [hunk 1]
```diff
@@ -638,6 +638,14 @@ class CreatePublication(InformationContentEntity):
         None,
         description="""The study abstract that describes the goals. Can also hold abstract from a publication related to this study.""",
     )
+    author: Optional[str] = Field(
```
Comment (sbilge) [inline metadata_transpiler_service/creation_models.py:641]: Are the creation model changes supposed to come from YAML artifacts? Does this addition require a change in the YAML?

Comment (galinatremper) [reply metadata_transpiler_service/creation_models.py:641]: Yes, they are.

**Review by sbilge — APPROVED** [review]

**Comment by sbilge** [comment]
**Added updated mapping.json (publication author, journal, year)**

- Closes GDEV-1148

**Pull request merged**